live/channel.go

```diff
@@ -29,11 +26,21 @@ type Channel struct {
 	Path string `json:"path,omitempty"`
 }
 
-// ParseChannel parses the parts from a channel ID:
-//   ${scope} / ${namespace} / ${path}.
-// Channel parts allowed to have alphanumeric symbols, underscore and dash.
+const (
+	maxChannelLength = 160
```

Comment:
should we make this 158?  since the first character(s) are actually the orgId/?

Reply:
I suppose it should work fine for 160 too, we still have 29 chars for internal purposes.